<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>objDec</class>
 <widget class="QWidget" name="objDec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>objDec</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="nn">
       <attribute name="title">
        <string>HOG+NN</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLabel" name="original">
            <property name="text">
             <string>original</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>pytorch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="gt_tl_x">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>gt top left x: </string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="gt_tl_y">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>gt top left y: </string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="gt_br_x">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>gt bottom right x: </string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="gt_br_y">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>gt bottom right y: </string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="L">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>seg size: </string>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>300</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="win_step">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>step: </string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="hog">
              <property name="prefix">
               <string>HOG size: </string>
              </property>
              <property name="maximum">
               <number>256</number>
              </property>
              <property name="value">
               <number>32</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="score_thre">
              <property name="prefix">
               <string>score threshold: </string>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.700000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="train_num">
              <property name="prefix">
               <string>training num: </string>
              </property>
              <property name="minimum">
               <number>200</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="lr">
              <property name="prefix">
               <string>learning rate: </string>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="crop_num">
              <property name="prefix">
               <string>crop num: </string>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="iou_th">
              <property name="prefix">
               <string>iou threshold: </string>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pick">
              <property name="text">
               <string>pick training</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="train">
              <property name="text">
               <string>train</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pick2">
              <property name="text">
               <string>pick destination</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="detect">
              <property name="text">
               <string>detect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="evaluation">
              <property name="text">
               <string>evaluate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="svm">
       <attribute name="title">
        <string>HOG+SVM</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="svm_pos_folder">
          <property name="text">
           <string>pick train pos folder</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="svm_neg_folder">
          <property name="text">
           <string>pick train neg folder</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="train_svm">
          <property name="text">
           <string>training</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="svm_dst">
          <property name="text">
           <string>pick destinate image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="run_svm">
          <property name="text">
           <string>run svm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QTextBrowser" name="svm_text"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pre">
       <attribute name="title">
        <string>PRE-TRAINED</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="pick_pretrained_img">
          <property name="text">
           <string>pick destinate image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="run_model">
          <property name="text">
           <string>runing model</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTextBrowser" name="pretrained_text"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="dst">
       <property name="text">
        <string>destination</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="Line" name="hline">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="result">
       <property name="text">
        <string>result</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
